src/image-components/ETVLogo.vue

```diff
@@ -0,0 +1,57 @@
+<template>
```

Comment:
Although this component isn't used yet, this sets up the Enjoy The Vue logo with text for use on the home page.

Reply:
Love it!